Tests/MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -118,11 +118,15 @@ class MapboxCoreNavigationTests: TestCase {
     
     func testNewStep() {
```

Comment:
`testNewStep` started failing once again due to mismatch of received spoken instructions. Had to increase number of coordinates to get missing instruction.